solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -158,4 +178,71 @@ mod benchmark_helper {
             RuntimeOrigin::from(pallet_xcm::Origin::Xcm(location))
         }
     }
+
+    impl snowbridge_pallet_inbound_queue::BenchmarkHelper<Runtime> for EthSystemBenchHelper {
+        fn initialize_storage(beacon_header: BeaconHeader, block_roots_root: H256) {
+            let submit_message = snowbridge_pallet_inbound_queue_fixtures::register_token::make_register_token_message();
+            let envelope: Envelope = Envelope::try_from(&submit_message.message.event_log).unwrap();
+
+            Channels::<Runtime>::set(
+                envelope.channel_id,
+                Some(Channel {
+                    agent_id: Default::default(),
+                    para_id: Default::default(),
+                }),
+            );
+
+            EthereumBeaconClient::store_finalized_header(beacon_header, block_roots_root).unwrap();
+        }
+    }
+
+    pub struct DoNothingMessageProcessor;
+
+    impl MessageProcessor for DoNothingMessageProcessor {
+        fn can_process_message(_: &Channel, _: &Envelope) -> bool {
+            true
+        }
+
+        fn process_message(_: Channel, _: Envelope) -> Result<(), sp_runtime::DispatchError> {
+            Ok(())
+        }
+    }
+}
+
+#[cfg(test)]
+mod test_helpers {
+    use snowbridge_core::inbound::{Log, Proof, VerificationError, Verifier};
+
+    pub struct MockVerifier;
+
+    impl Verifier for MockVerifier {
+        fn verify(_: &Log, _: &Proof) -> Result<(), VerificationError> {
+            Ok(())
+        }
+    }
+}
+
+impl snowbridge_pallet_inbound_queue::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    #[cfg(not(test))]
+    type Verifier = EthereumBeaconClient;
+    #[cfg(test)]
+    type Verifier = test_helpers::MockVerifier;
+    type Token = Balances;
+    type XcmSender = DoNothingRouter;
+    type GatewayAddress = EthereumGatewayAddress;
+    type MessageConverter = DoNothingConvertMessage;
+    type ChannelLookup = EthereumSystem;
+    type PricingParameters = EthereumSystem;
+    type WeightInfo = weights::snowbridge_pallet_inbound_queue::SubstrateWeight<Runtime>;
+    #[cfg(feature = "runtime-benchmarks")]
+    type Helper = benchmark_helper::EthSystemBenchHelper;
+    type WeightToFee = WeightToFee;
+    type LengthToFee = ConstantMultiplier<Balance, TransactionByteFee>;
+    type MaxMessageSize = ConstU32<2048>;
```

Comment:
Isnt this `maxMessageSize` limiting us somewhat? it is the same comment I had before, mainly 2048/32 only allows for 32 validators being injected. What are the implications of increasing it?

Reply:
I think it is only for fee estimation, but plz confirm

Reply:
And if so, please add a todo here for us to properly set this value

Reply:
It is only used if size of xcmp message constructed out of the incoming message exceeds the max message size. So, it does not affect us. 

I will add a todo still.